src/olympia/abuse/models.py

```diff
@@ -424,19 +440,27 @@ def report(self):
         abuse = self.abuse_report
         reporter = self.get_cinder_reporter()
         reason = AbuseReport.REASONS.for_value(abuse.reason)
-        job_id = self.get_helper().report(
+        job_id = self.get_entity_helper().report(
             report_text=abuse.message, category=reason.api_value, reporter=reporter
         )
         self.update(job_id=job_id)
 
+    def process_decision(self, *, decision_id, decision_date, decision_actions):
+        self.update(
+            decision_id=decision_id,
+            decision_date=decision_date,
+            **({'decision_action': decision_actions[0]} if decision_actions else {}),
```

Comment:
We should probably complain loudly (raise an exception ? It will be shown in both Cinder and Sentry, which I think is what we want ?) if there are no action, an unknown action not in `DECISION_ACTIONS`, or more than one action.

Reply:
I'll handle this in https://github.com/mozilla/addons-server/issues/21402